docs/pubsub-client.rst

```diff
@@ -1,20 +1,13 @@
-.. toctree::
-  :maxdepth: 1
-  :hidden:
-
-Pub/Sub
--------
-
 Client
```

Comment:
ISTM that the page should refer somehow to the package it is documenting (maybe `Pub/Sub Client`?)  Having to look to the URL bar is the Wrong Thing (TM).